Composer/ScriptHandler.php

```diff
@@ -254,33 +254,18 @@ public static function doBuildBootstrap($bootstrapDir)
             'Symfony\\Component\\HttpFoundation\\FileBag',
             'Symfony\\Component\\HttpFoundation\\ServerBag',
             'Symfony\\Component\\HttpFoundation\\Request',
-            'Symfony\\Component\\HttpFoundation\\Response',
-            'Symfony\\Component\\HttpFoundation\\ResponseHeaderBag',
 
-            'Symfony\\Component\\DependencyInjection\\ContainerAwareInterface',
-            'Symfony\\Component\\DependencyInjection\\Container',
             'Symfony\\Component\\HttpKernel\\Kernel',
             'Symfony\\Component\\ClassLoader\\ClassCollectionLoader',
             'Symfony\\Component\\ClassLoader\\ApcClassLoader',
-            'Symfony\\Component\\HttpKernel\\Bundle\\Bundle',
```

Comment:
Nope: the kernel does this in two stages. First generate classes.map, then classes.php. This it works well.